File: query_feature_213.py
import sqlite3
import json

db_path = 'features.db'
conn = sqlite3.connect(db_path)
cursor = conn.cursor()

cursor.execute('''
    SELECT id, priority, category, name, description, steps, passes, in_progress, dependencies
    FROM features
    WHERE id = 213
''')

row = cursor.fetchone()
if row:
    result = {
        'id': row[0],
        'priority': row[1],
        'category': row[2],
        'name': row[3],
        'description': row[4],
        'steps': row[5],
        'passes': row[6],
        'in_progress': row[7],
        'dependencies': row[8]
    }
    print(json.dumps(result, indent=2))
else:
    print('Feature #213 not found')

conn.close()

File: query_feature_213.js
const sqlite3 = require('sqlite3').verbose();
const path = require('path');

const dbPath = path.join(__dirname, 'features.db');
const db = new sqlite3.Database(dbPath);

const sql = `SELECT id, priority, category, name, description, steps, passes, in_progress, dependencies
              FROM features
              WHERE id = 213`;

db.get(sql, [], (err, row) => {
  if (err) {
    console.error('Error:', err.message);
    process.exit(1);
  }
  if (row) {
    console.log(JSON.stringify(row, null, 2));
  } else {
    console.log('Feature #213 not found');
  }
  db.close();
});
